src/viam/components/base/service.py

```diff
@@ -111,3 +124,6 @@ async def DoCommand(self, stream: Stream[DoCommandRequest, DoCommandResponse]) -
         result = await base.do_command(command=struct_to_dict(request.command), timeout=timeout, metadata=stream.metadata)
         response = DoCommandResponse(result=dict_to_struct(result))
         await stream.send_message(response)
+
+    async def GetGeometries(self, stream: Stream[GetGeometriesRequest, GetGeometriesResponse]) -> None:
+        raise MethodNotImplementedError("GetGeometries").grpc_error
```

Comment:
Do all the services not need `GetKinematics`, too?

Reply:
No, GetKinematics is only used by the `arm` component.